apps/frontend/src/components/ui/location.tsx

```diff
@@ -0,0 +1,27 @@
+import { useGeolocated } from "react-geolocated";
+
+export const LocationDemo = () => {
+    const { coords, isGeolocationAvailable, isGeolocationEnabled } =
+        useGeolocated({
+            positionOptions: {
+                enableHighAccuracy: true,
+            },
+            userDecisionTimeout: 5000,
+            watchPosition: true,
+        });
+
+    return !isGeolocationAvailable ? (
+        <div>Your browser does not support Geolocation</div>
```

Comment:
This is fine for the demo, but I think if geolocation isn't available we shouldn't lock someone out of the page, for future reference